src/pages/mobile/AppBrowser.vue

```diff
@@ -102,7 +103,16 @@ export default {
       },
     }),
   },
-  mounted() {
+  async mounted() {
+    const sdk = this.$store.state.sdk.then ? await this.$store.state.sdk : this.$store.state.sdk;
+
+    const connection = BrowserWindowMessageConnection({ target: this.$refs.iframe.contentWindow });
+    sdk.addRpcClient(connection);
```

Comment:
don't you forget to remove the connection on component destruction? I think it may cause a memory leak, somewhere sdk has already implemented a method for that.